src/components/Api.js

```diff
@@ -0,0 +1,105 @@
+import { data } from "autoprefixer";
+
+export class Api {
+  constructor() {
+    // constructor body
+    this._baseUrl = "https://around-api.en.tripleten-services.com/v1";
+    this._headers = {
+      authorization: "2e65c592-5cc5-4cb0-a6bf-23fa612e6f57",
+      "Content-Type": "application/json",
+    };
```

Comment:
Please, don't hardcode this data inside `Api`

According to the task you need to pass full `headers` and `baseUrl` into the **constructor** of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ .